build.gradle

```diff
@@ -397,7 +397,7 @@ task listDist(type: Exec) {
 }
 
 task distRuntime(type: Copy) {
-    dependsOn "${hostName}CrossDistRuntime"
+    dependsOn "${hostName}CrossDistRuntimeAndCache"
```

Comment:
This dependency seems inconsistent. Currently just `distRuntime` builds caches while `crossDistRuntime` and `<target>CrossDistRuntime` don't.